og.module

```diff
@@ -71,9 +71,11 @@ function og_entity_update(EntityInterface $entity) {
  * Implements hook_entity_predelete().
  */
 function og_entity_predelete(EntityInterface $entity) {
+  /** @var \Drupal\og\MembershipManagerInterface $membership_manager */
```

Comment:
I'd put this later on, just before the foreach.

Reply:
But then I would have to get it twice.